web/app_options.js

```diff
@@ -55,10 +55,19 @@ const OptionKind = {
   API: 0x04,
   WORKER: 0x08,
   EVENT_DISPATCH: 0x10,
-  UNDEF_ALLOWED: 0x20,
   PREFERENCE: 0x80,
 };
 
+// These fields *must* be lowercase to match the `typeof` operator,
```

Comment:
Alternatively we could also use `(typeof value).toUpperCase()` below instead. This allows us to remove the comment and use uppercased enum keys as usual.

Reply:
That's a really good suggestion, since the inconsistency probably isn't great; thank you!